frontend/src/modules/member/components/list/member-list-toolbar.vue

```diff
@@ -68,6 +68,13 @@
           />
           {{ markAsTeamMemberOptions.copy }}
         </el-dropdown-item>
+        <el-dropdown-item
+          :command="{ action: 'editAttribute' }"
+          :disabled="isEditLockedForSampleData"
+        >
+          <i class="ri-lg ri-file-edit-line mr-1" />
+          Edit attribute
```

Comment:
Maybe to be consistent with the other dropdown actions it could be "Edit attributes"

Reply:
Good catch! However, the Figma designs referenced 'Edit Attribute'. Nuno also suggested updating the copy to better fit the context, as we support bulk editing of a single attribute.